### PR TITLE
Outer Rim Galaxies patch

### PR DESCRIPTION
**Pull request by Person243546**
## Additions

Adds a compatibility patch for [O21] Outer Rim Galaxies [DevBuild].

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Weapons.xml [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+				<value>
+					<equippedStatOffsets>
+						<MeleeCritChance>0.4</MeleeCritChance>
+						<MeleeParryChance>0.7</MeleeParryChance>
+						<MeleeDodgeChance>0.2</MeleeDodgeChance>	
+					</equippedStatOffsets>
+				</value>
+			</li>
+			
+			<!-- Lightsaber patch from fully functional lightsaber mod -->
```
Comment (N7Huntsman) [inline Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Weapons.xml:513]: One of these lightsaber operation is failing. I'm not sure specifically which.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Please change the patch folder name from [O21] to O21 if you would, so that it'll alphabetize with the other O21 mods.

**Comment by N7Huntsman** [comment]
The version of the mod on the workshop is still 1.2, and it's unclear what its status going forward will be.

File: Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Weapons.xml [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+							  <armorPenetrationBlunt>0.555</armorPenetrationBlunt>
+							  <linkedBodyPartsGroup>Muzzle</linkedBodyPartsGroup>
+							</li>
+						  </tools>
+						</value>
+					  </li>
+
+				  <li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[		
+					defName="Amban_Rifle"
```
Comment (N7Huntsman) [inline Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Weapons.xml:50]: This was named to `O21_OR_AmbanSniperRifle,` swap out references to it.

File: Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Apparel.xml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+		<match Class="PatchOperationSequence">
+		<operations>
+		
+			<li Class="PatchOperationConditional">
+				<xpath>Defs/ThingDef[defName="O21_OR_AmbanSniperRifle"]</xpath>
+				<match Class="PatchOperationSequence">
+					<operations>
+					
+					<li Class="PatchOperationReplace">
+						<xpath>Defs/ThingDef[defName="O21_Mando_BasicArmor"]/statBases/ArmorRating_Sharp</xpath>
```
Comment (N7Huntsman) [inline Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Apparel.xml:17]: This became, essentially, `O21_OR_MandalorianChest_Simple`. There was also a helmet and set of pauldrons added.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

File: Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Weapons.xml (hunk 1 above)
Comment (Person243546) [reply Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Weapons.xml:513]: Should be fixed now

File: Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Apparel.xml (hunk 3 above)
Comment (Person243546) [reply Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Apparel.xml]: Done

File: Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Weapons.xml (hunk 2 above)
Comment (Person243546) [reply Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Weapons.xml]: Done

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
No errors with the latest updates. All that remains are some various helmets for Clones, Mandalorians, etc. Where possible, I recommend patching the apparel bases rather than patching it one by itself.

**Comment by Person243546** [comment]
The patch already targets the helmet's base. I think all of the helmets are patched.

**Comment by Person243546** [comment]
I put Outer Rim Galaxies above CE in my load order

**Review by N7Huntsman — APPROVED** [review]
Seems to work.

**Pull request merged**